massa-models/src/api.rs

```diff
@@ -173,7 +174,12 @@ impl std::fmt::Display for AddressInfo {
         writeln!(f, "Address: {}", self.address)?;
         writeln!(f, "Thread: {}", self.thread)?;
         writeln!(f, "Sequential balance:\n{}", self.ledger_info)?;
-        writeln!(f, "Parallel balance:\n{}", self.sce_ledger_info)?;
+        writeln!(f, "Final Parallel balance:\n{}", self.final_sce_ledger_info)?;
+        writeln!(
+            f,
+            "Candidate Parallel balance:\n{}",
```

Comment:
As a user when I started to use the client I didn't understand the difference between candidate and final. Now that we split more of information between this two category it will be nice to have a line to explain when making a command that returning final and candidate things.

Reply:
Where do you expect that kind of doc ?

Reply:
At the top of the output of the command that display candidate and final things for example wallet info.

Reply:
And in the get_addresses too ?

Maybe we should have a call to discuss what concepts need some documentation and add a vocabulary/FAQ/key concepts command that would answer these questions ? I feel like having it displayed every time is a little overkill

Or display it only the first time the command is executed ?

Reply:
I agree that it is more and more confusing as it is concepts that do not exist in other blockchains. Displaying this information all the time is a bit overkill tho'.

In the long run command line client will probably mostly be used to manage nodes on servers so it needs to be practical not necessarily extremely user friendly. I think what we are mostly lacking is high level documentation on the wiki.

Reply:
the problem is that the new people don't look at the wiki after they successfully stake. But yeah if they are interested by understanding the terms it can be in the wiki because not everyone is interested.

Reply:
#2398